includes/setup-wizard.php

```diff
@@ -167,7 +167,18 @@ public function setup_wizard_header() {
             <title><?php esc_html_e( 'WPUF &rsaquo; Setup Wizard', 'wp-user-frontend' ); ?></title>
             <?php wp_print_scripts( 'wpuf-setup' ); ?>
             <?php do_action( 'admin_print_styles' ); ?>
-            <?php do_action( 'admin_head' ); ?>
+            <?php
+                global $current_screen;
+
+                if ( empty( $current_screen ) ) {
+
+                    if ( function_exists('set_current_screen') ) {
```

Comment:
this should be `if ( empty( $current_screen ) && function_exists( 'set_current_screen' ) )`